package-lock.json

```diff
@@ -0,0 +1,13032 @@
+{
+  "name": "docs",
```

Comment:
is it necessary to add to the PR?

Reply:
No - i can remove - was a consequence of running `npm install` :D